packages/element-lib/src/sidetree/batchRequests.spec.js

```diff
@@ -20,6 +19,7 @@ afterEach(async () => {
 
 describe('batchRequests', () => {
   it('can batch once..', async (done) => {
+    sidetree = await getLocalSidetree('batchRequestsOnce');
```

Comment:
These DB names were caused by filesystem contension from leveldown, its possible that using in memory has solved this problem and we don't need to use this argument anymore.